pkgs/os-specific/linux/nfs-utils/default.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
       "--with-systemd=${placeholder "out"}/etc/systemd/system"
       "--enable-libmount-mount"
       "--with-pluginpath=${placeholder "lib"}/lib/libnfsidmap" # this installs libnfsidmap
-      "--with-rpcgen=${rpcsvc-proto}/bin/rpcgen"
+      "--with-rpcgen=${rpcsvc-proto.nativeDrv or rpcsvc-proto}/bin/rpcgen"
```

Comment:
```suggestion
      "--with-rpcgen=${buildPackages.rpcsvc-proto}/bin/rpcgen"
```

Reply:
Ooh, didn't know you can do that. Thank you!